zeppelin-interpreter/src/test/java/org/apache/zeppelin/dep/DependencyResolverTest.java

```diff
@@ -87,10 +87,11 @@ public void testLoad() throws Exception {
     FileUtils.cleanDirectory(testCopyPath);
 
     // load from added repository
-    resolver.addRepo("sonatype",
-        "https://oss.sonatype.org/content/repositories/agimatec-releases/", false);
-    resolver.load("com.agimatec:agimatec-validation:0.12.0", testCopyPath);
-    assertEquals(testCopyPath.list().length, 8);
+    // temporarily disable until the below repo is back up
```

Comment:
Can we change to another repo instead of disable it ? Because I guess we may never get it back after we disable it.

Reply:
any recommendation?

Reply:
I checked this repo. This repo is not down, it is the artifact not accessible now.

Reply:
Sure I understand, let me update this.